src/components/views/elements/ActionButton.tsx

```diff
@@ -88,7 +92,9 @@ export default class ActionButton extends React.Component<IProps, IState> {
                 className={classNames.join(" ")}
                 onClick={this.onClick}
                 onMouseEnter={this.onMouseEnter}
-                onMouseLeave={this.onMouseLeave}
+                onMouseLeave={this.onHideTooltip}
+                onFocus={this.onShowTooltip}
+                onBlur={this.onHideTooltip}
```

Comment:
These ones might be better off just called, "showTooltip", ie. they're not handling the the 'show tooltip' event so `onShowTooltip` is a bit misleading.